rkt/image/filefetcher.go

```diff
@@ -17,9 +17,12 @@ package image
 import (
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 
+	"golang.org/x/crypto/sha3"
```

Comment:
Consider using something more commonly used, it was not easy to get something to verify an SHA-3 on my bleeding-edge distro.